src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -375,125 +379,128 @@ private static final class EnumMap {
     @SuppressWarnings("removal")
     private static MethodHandle generateInnerClass(MethodHandles.Lookup caller, Object[] labels) {
         List<EnumDesc<?>> enumDescs = new ArrayList<>();
+        List<Class<?>> extraClassLabels = new ArrayList<>();
 
         byte[] classBytes = Classfile.of().build(ClassDesc.of(typeSwitchClassName(caller.lookupClass())), clb -> {
             clb.withFlags(AccessFlag.FINAL, AccessFlag.SYNTHETIC)
```

Comment:
`AccessFlag.SUPER` is missing, this will make this class a value class in the Valhalla repo